### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `codespell -S ./assets -L als,trough`

### CODE REVIEW

File: assets/colorcet.m [hunk 1]
```diff
@@ -9073,7 +9073,7 @@
 
 case { 'C1'  'C01' }
 descriptorname = 'cyclic_mrybm_35-75_c68_n256';
-description = 'Cyclic: magenta - red - yellow - blue. Alows four orientations/phase angles to be visulaised.';
+description = 'Cyclic: magenta - red - yellow - blue. Allows four orientations/phase angles to be visulaised.';
```
Comment (jbednar) [inline assets/colorcet.m:9076]: Please remove any fixes to the contents of `assets`; these files are mirrors of upstream content whose source is located elsewhere, and we do not want those files to diverge from their source files. Ideally such fixes would be filed at the upstream repo, though I'm not sure where that is (we get the files from https://colorcet.com).

File: colorcet/sineramp.py [hunk 2]
```diff
@@ -80,7 +80,7 @@
 visual performance of the viewer.
 
 The Default Amplitude:
-This is set at 12.5 so that from peak to trough we have a local feature of
+This is set at 12.5 so that from peak to through we have a local feature of
```
Comment (jbednar) [inline colorcet/sineramp.py:83]:
```suggestion
This is set at 12.5 so that from peak to trough we have a local feature of
```

(The original is correct.)

**Review by jbednar — CHANGES_REQUESTED** [review]
Thanks! Some of these are good fixes, and this should be mergeable when it has only those fixes.

**Review by jbednar — APPROVED** [review]
Perfect, thanks!

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**